### DISCUSSION TITLE
Is this error correct or I'm missing something

### DISCUSSION THREAD

**Discussion by babakfp** [pocketbase/pocketbase#2817] [Q&A] [answered]
![Screenshot 2023-06-30 073856](https://github.com/pocketbase/pocketbase/assets/44144724/91133783-3877-4128-bcd2-382ecf01c2a9)

Leaving it blank or writing `0` is the same, right?🤔

**Comment by ganigeorgiev** [comment, marked as answer]
> Leaving it blank or writing 0 is the same, right?

No. The underlying type is a pointer and it differentiate between empty (no restriction) and a filled input value. The error message is a little misleading in this case. I'll consider changing it or adding an exception for 0.

**Reply by ganigeorgiev** [reply]
_I've pushed in `develop` a small fix that removes the blank validator allowing the use of `0` as min value._
